server_admin/topics/clients/con-client-scopes.adoc

```diff
@@ -98,7 +98,7 @@ Client scope::
 Client scopes are entities in {project_name} that are configured at the realm level and can be linked to clients. Client scopes are referenced by their name when a request is sent to the {project_name} authorization endpoint with a corresponding value of the *scope* parameter. See the <<_client_scopes_linking, client scopes linking>> section for more details.
 
 Role scope mapping::
-Available under the *Scope* tab of a client or client scope. Role scope mapping allows you to limit the roles which can be used in the access tokens. See the <<_role_scope_mappings, Role Scope Mappings section>> for more details.
+This is available under the *Scope* tab of a client or client scope. Use *Role scope mapping* to limit the roles which can be used in the access tokens. See the <<_role_scope_mappings, Role Scope Mappings section>> for more details.
```

Comment:
Use "that" not "which" when the clause that follows qualifies the meaning of the first clause.
So "to limit the roles that..."